examples/ansible/complex_vault_test.yaml

```diff
@@ -1,10 +1,13 @@
 $ANSIBLE_VAULT;1.1;AES256
-39326637626638343730653837323833393866393562643366353061636338323862383336623637
-3463353136613032383465393830306661643464636564610a353136663166623939353339346337
-35666131323030303230633634646432353664653961623135303538633830373361656430353934
-6135373030653536330a323738626531653036636132643931643739653038623565396666653833
-61653763663137656533353433303362383036663738366561313836386634636535636332346331
-30343964343137383439313738376431373037363830333936316530386663616361323866343833
-34633735336333333735393061666432386137346363356662303763623865373531633834396630
-39343533363037616536653462656566383935616666376261366165323330333532613033626362
-33643831313634303032313761646161623336346638313361363762383933356331
+39643034383037333236643563633139393630663361393838303862313639326336663739356161
+3436373839303865326564393536633332623434313063330a356633633634613636626466326362
+38383231653335313464363235616265616361316537303639613937356635316135366335316131
+3130393332383661340a356232366431343766393733326335363236353062373766363065633965
+64646630653133393764616466386236336261373938356165303033363237363734386630343338
+64623739633131303032663839316165643339393434393662313838396361346336663831383738
+38393866373638653632613666636233366563343031376438333335303630336338623434396538
+65646631633639613234313833343564376332346264653639386261373930303566323261613764
+38633763656662306535343866643833303931613762373761386266316364323837333066616336
+36316665333731336638333134613730643632336535653336316366333466356363346462333139
+38323437623937653436343934353263396239643833373836373633323362636362343636366538
+65623761656431646136
```

Comment:
Instead of starting to add nested dict here, can you add example with bool in `sanitized_vault.yml`, to match your code changes, and put this modification in your next PR ?

Reply:
I will.